### PR TITLE
add artificial-adversary

### PR DESCRIPTION
**Pull request by rluria14**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [ ] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by rluria14** [comment]
@oblute @ndmaxar agree to be maintainers?

File: recipes/artificial-adversary/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/airbnb/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: e086e63701f8df5978f08d35f2dd0bfb486241fd92925cae76c0c59ce6adc990
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv "
```
Comment (scopatz) [inline recipes/artificial-adversary/meta.yaml:14]: This can be `noarch: python`

Comment (rluria14) [reply recipes/artificial-adversary/meta.yaml:15]: done, thanks

**Comment by rluria14** [comment]
@conda-forge-admin please restart ci

**Comment by rluria14** [comment]
@conda-forge-admin please restart ci

**Comment by rluria14** [comment]
problems with appveyor???

**Comment by scopatz** [comment]
> problems with appveyor???

No need to worry about that

**Comment by rluria14** [comment]
@scopatz think this one is good to go then if the appveyor failure isn't an issue

**Comment by scopatz** [comment]
Waiting on @ndmaxar :wink:

**Comment by rluria14** [comment]
@scopatz @conda-forge/staged-recipes this recipe is ready for final review and merge

**Comment by scopatz** [comment]
Thanks!

**Pull request merged**